src/renderer/views/wallet/Interact/BondView.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useCallback, useMemo, useState } from 'react'
+
+import * as RD from '@devexperts/remote-data-ts'
+import { AssetRuneNative, BaseAmount, baseToAsset } from '@xchainjs/xchain-util'
+import * as A from 'fp-ts/Array'
+import * as FP from 'fp-ts/function'
+import * as O from 'fp-ts/Option'
+import { useObservableState } from 'observable-hooks'
+import { useIntl } from 'react-intl'
+import * as RxOp from 'rxjs/operators'
+
+import { Bond } from '../../../components/interact/forms'
+import { Button } from '../../../components/uielements/button'
+import { ZERO_ASSET_AMOUNT } from '../../../const'
+import { useThorchainContext } from '../../../contexts/ThorchainContext'
+import { useWalletContext } from '../../../contexts/WalletContext'
+import { eqAsset } from '../../../helpers/fp/eq'
+import { INITIAL_INTERACT_STATE } from '../../../services/thorchain/const'
+import { InteractState } from '../../../services/thorchain/types'
+import * as Styled from './InteractView.styles'
+
+type Props = {
+  walletAddress: string
+}
+
+export const BondView: React.FC<Props> = ({ walletAddress }) => {
+  const { balancesState$ } = useWalletContext()
+  const [interactState, setInteractState] = useState<InteractState>(INITIAL_INTERACT_STATE)
+  const { interact$ } = useThorchainContext()
+  const intl = useIntl()
+
+  const [runeBalance] = useObservableState(
+    () =>
+      FP.pipe(
+        balancesState$,
+        RxOp.map(({ balances }) => balances),
+        RxOp.map(
+          FP.flow(
+            O.chain(
+              A.findFirst(
+                ({ walletAddress: balanceWalletAddress, asset }) =>
+                  balanceWalletAddress === walletAddress && eqAsset.equals(asset, AssetRuneNative)
+              )
+            ),
+            O.map(({ amount }) => amount),
+            O.map(baseToAsset),
+            O.getOrElse(() => ZERO_ASSET_AMOUNT)
+          )
+        )
+      ),
+    ZERO_ASSET_AMOUNT
+  )
+
+  const bondTx = useCallback(
+    ({ amount, memo }: { amount: BaseAmount; memo: string }) => interact$({ amount, memo }).subscribe(setInteractState),
```

Comment:
Subscription to `interact$` should be unsubscribed before component will be unmounted and before calling `boundTx` for next time.